ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -1202,3 +1202,13 @@ literals:
 - thermal_energy_accumulator
 - ground_isolation_command
 - ground_temperature_sensor
+
+## New Fields for PAC (ACU) Units
+
+- discharge_fan_run_time_accumulator
+- compressor_run_time_accumulator
+- evaporative_fan_speed_sensor
+- condensing_fan_speed_percentage_command
+- zone_air_override_temperature_status
```

Comment:
is this covered by `user_occupancy_override_status`? see the [AHAC type](https://github.com/google/digitalbuildings/blob/f40d6aa06cab3a28f1a041705297c247c99f936b/ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml#L3972:).